### ISSUE TITLE
Container support for Windows 10 IoT Enterprise LTSC 2021

### ISSUE THREAD

**Issue by krotz-dieter** [microsoft/Windows-Containers#249]
**Is your feature request related to a problem? Please describe.**
We need base container images  for Windows 10 IoT Enterprise LTSC 2021 !
Currently we use 20H2 where we still can develop windows containers , this is ticking bomb which will hit our company (>1000 developers).
We cannot jump to Windows 11 and we cannot upgrade the systems with Windows Server licenses due to higher costs.
We need images for 21H2, the strategy described in:
https://techcommunity.microsoft.com/t5/containers/windows-server-2022-and-beyond-for-containers/ba-p/2712487
is a huge problem for us. In my eyes that's a breaking behavior, it was always stated from Microsoft side that it is allowed not only to develop and test container workloads on Windows 10 IoT, actually also production workloads are supported on Windows 10 IoT.

Hyper-V isolation is not an solution, it's extremely slow and is not supported for example by containerd.

**Describe the solution you'd like**
Base Images which allow process isolation on 21H2 for Windows 10 IoT Enterprise LTSC 2021, 
basically the ability to host .NET Framework services as well as windows apps in containers.

**Describe alternatives you've considered**
Moving all our services from Windows containers to Linux containers. 
This is a very costly operation since we have millions of line of code to adapt and rewrite.

**Additional context**
Medical Industry Systems available at customer site

**Comment by fady-azmy-msft** [comment]
Hey @SyngoPredevelopment thank you for writing to us. Can you clarify what is the issue with 21H2?

**Comment by krotz-dieter** [comment]
The problem is that in Windows 10 21H2, 21H1 you don't have base images for windows containers whic supports process isolation.
Also here it is stated:
https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility?tabs=windows-server-2022%2Cwindows-10-21H1
that the images don't support process isolation. For mcr.microsoft.com/windows/servercore images you don't have base images for 21H2 for example, you jumped directly to Windows 11. We have a major problem in that, we cannot use Windows 21H2 at all in that case.

**Comment by krotz-dieter** [comment]
To be more concrete, we need to run containers with dotnet framework apps in process isolation on Windows 21H2, the LTSC version for windows 10 does not support process isolation for OS build 19044.1826 which is the latest Win 10 LTSC version. The are no base images for mcr.microsoft.com/dotnet/framework/runtime:4.8 (which is actually based on the server core images, there 21H2 where omitted) which can give us the needed functionality. This is very much needed in our case !

**Comment by fady-azmy-msft** [comment]
Understood and thanks for sharing. Let me look into this and get back to you.

**Comment by krotz-dieter** [comment]
Many thanks

**Comment by fady-azmy-msft** [comment]
Wanted to let you know that we're still looking into this but I don't have an update to share at the moment.

**Comment by krotz-dieter** [comment]
We have found an workaround how to use the Windows 10 20H2 containers also on 21H2 and 22H2.

**Comment by suhancz** [comment]
> We have found an workaround how to use the Windows 10 20H2 containers also on 21H2 and 22H2.

Can you please share the workaround with the community
